business/namespaces.go

```diff
@@ -129,12 +127,110 @@ func (in *NamespaceService) GetNamespaces(ctx context.Context) ([]models.Namespa
 		labelSelectorExcludeValue = excludeLabelList[1]
 	}
 
+	namespaces := []models.Namespace{}
+	// Merge namespaces from different clusters
+	// Namespace sameness
+	nsmap := make(map[string]models.Namespace)
+
+	if clientFactory == nil {
+		var err error
+		clientFactory, err = kubernetes.GetClientFactory()
+		if err != nil {
+			log.Errorf("Error getting client factory: %v", err.Error())
+			return nil, fmt.Errorf("Error getting client factory")
+		}
+	}
+
+	var clusterError error
+	for _, cluster := range clientFactory.GetClusterNames() {
+		kialiClient := clientFactory.GetSAClient(cluster)
+		nsList, error := in.GetNamespacesByClient(kialiClient, cluster)
```

Comment:
Here, maybe in the future we need to fetch the information in go routines and then do the merging.

Reply:
Thanks for the suggestion! I think is a good idea, I have made a commit with the changes.